src/app/shared/services/claims-facade/claims-facade.service.ts

```diff
@@ -176,6 +176,13 @@ export class ClaimsFacadeService {
               from(this.setIsRevokedOffChainStatus(enrolment))
             ),
           ])
+        ),
+        switchMap((enrolments: EnrolmentClaim[]) =>
+          forkJoin([
+            ...enrolments.map((enrolment) =>
+              from(this.setDecodedToken(enrolment))
```

Comment:
Good idea to decode it when creating enrolment-claim object. 👍 We could decode both: token and issuedToken instead of doing it in view-requests component by using `TokenDecodeService`. But this doesn't need to be done now, we can refactor it next time :)